source/_docs/pantheon-yml.md

```diff
@@ -36,6 +36,36 @@ protected_web_paths:
 * You may not be able to protect files or paths with special characters
 * Wait a few seconds for changes to take effect
 
+### Enforce HTTPS + HSTS
+
+HTTPS adds a layer of encryption that prevents others from snooping on or tampering with traffic to your site. HTTP Strict Transport Security (**HSTS**) instructs browsers to only connect via HTTPS and helps protect websites against protocol downgrade attacks and cookie hijacking.
+
+<div class="alert alert-info" role="alert">
+<h4 class="info">Note</h4>
+<p markdown="1">Before adjusting `enforce_https`, review and understand the configuration options and all considerations to avoid unintended consequences.</p>
+</div>
+
+Ensure that your site will always use HTTPS to deliver content with `enforce_https`. Five values are available, from least to most secure:
+
+- `off`
+- `transitional` - Redirect to HTTPS with a 5-minute HSTS header
+- `transitional+subdomains` Redirect to HTTPS with a 5-minute HSTS header that will enforce HTTPS for subdomains, even those not on Pantheon.
+- `full` - Redirect to HTTPS with a year-long HSTS header
+- `full+subdomains` - Redirect to HTTPS with a year-long HSTS header that will enforce HTTPS for subdomains, even those not on Pantheon.
+   - This is the recommended and most secure configuration.
+
+#### Considerations
+* Use of `full` or `full+subdomains` should be treated as a commitment. If your site is unable to serve HTTPS (e.g. by moving to a host that doesn't support HTTPS) after sending a long-duration HSTS header, visitors will be unable to access your site.
+* Any option with `+subdomains` should only be used if you want to enforce HTTPS for *all subdomains, even those not connected to Pantheon*.
+* To prepare your site to serve all content via HTTPS, follow the [Switching Sites from HTTP to HTTPS](/docs/http-to-https/) doc.
+
+#### Test Your Site's HSTS Configuration for an A+ Rating
```

Comment:
Most other sections are much shorter, and link out to a fuller doc page with additional context (e.g., Drush, PHP, Nested Docroot, etc.) 

Is this section better served by `/docs/http-to-https` ?